README.md

```diff
@@ -54,6 +54,28 @@ http_archive(
 )
 ```
 
+To register a hermetic Python toolchain, you can add to the `WORKSPACE` file:
+
+```python
+load("@rules_python//python:repositories.bzl", "python_register_toolchains")
+
+# Multiple versions are available in the @rules_python//python:versions.bzl file.
+python_register_toolchains(
+    name = "python310",
+    python_version = "3.10",
+)
+
+load("@python310_host//:defs.bzl", "interpreter")
```

Comment:
I kinda find this confusing that it's called `host` toolchain. It's resolved for the host platform, but it's not the hosts interpreter.

Reply:
what would you want to name it? this one matches rules_nodejs

Reply:
Any suggestions for better naming?

Reply:
I liked the `resolved` naming scheme, matches the `resolved_toolchain`?
https://github.com/bazelbuild/rules_nodejs/blob/9dddd19bc9bc91cb5a0fd932e3cc2d601133a9ea/nodejs/private/toolchains_repo.bzl

I know this isn't the toolchain, but `resolved_interpreter` or similar?

Reply:
Addressed in [7eacdf1](https://github.com/bazelbuild/rules_python/pull/618/commits/7eacdf1eb22c1f24df10d332c0908413c21f30f4).

Reply:
I've opened https://github.com/bazelbuild/rules_python/pull/656 which relates to this discussion.